widgetssdk/build.gradle

```diff
@@ -62,4 +62,30 @@ dependencies {
     // removed because becomes slow on Android 11. They say it is fixed, but I still have the issue.
     // https://square.github.io/leakcanary/changelog/
     // debugImplementation "com.squareup.leakcanary:leakcanary-android:$leakCanaryVersion"
+
+    // Replaced with a local AAR file
+//    implementation "com.glia:android-sdk:$gliaSdkVersion"
+    // Glia sdk aar
+    implementation(name: 'glia-core-sdk-0.20.0', ext: 'aar')
```

Comment:
Mb better to use variable use have declared before? $gliaSdkVersion

Reply:
Fixed: https://github.com/salemove/android-sdk-widgets/compare/7734b2d3066197a01a15942ea951e793a6c34266..a27372cd5fbfc29655174516012ff7ea88e9471e